src/Reflection/ReflectionAttribute.php

```diff
@@ -11,19 +11,31 @@
 use Roave\BetterReflection\Reflection\StringCast\ReflectionAttributeStringCast;
 use Roave\BetterReflection\Reflector\Reflector;
 
+use function array_map;
+
 class ReflectionAttribute
 {
+    private string $name;
+
+    /** @var array<int|string, Node\Expr> */
```

Comment:
Is mixing `int` and `string` arguments allowed?

Wondering if this could be `array<int, Node\Expr>|array<string, Node\Expr>` :thinking:

Reply:
I think `array<int|string, Node\Expr>` is the most accurate for code like this:

```
#[Attr('arg1', 'arg2', arg3: self::class, arg4: [0, 1]])]
```

Reply:
Works for me :+1: